DemoApp/Info.plist

```diff
@@ -36,9 +36,21 @@
 				<string>streamvideo</string>
 			</array>
 		</dict>
+		<dict>
+			<key>CFBundleTypeRole</key>
+			<string>Editor</string>
+			<key>CFBundleURLName</key>
+			<string>com.googleusercontent.apps.347024607410-48j4atipav0tcr4pesap4elr1u9t11uh</string>
```

Comment:
Probably this one should be hidden from public, right? We can use build variables and inject it when releasing a new build

Reply:
How do we do that? Do we have an example to follow?

Reply:
We can have it as a build variable in CI environment (in the same way we are using some credentials for testing). @testableapple  can help probably